app/mailers/app_mailer.rb

```diff
@@ -10,4 +12,11 @@ def send_forgot_password(user)
          from: "info@myflix.com",
          subject: "Please reset your password"
   end
+
+  def send_invitation_email(invitation)
+    @invitation = invitation
+    mail to: @invitation.recipient_email,
+         from: "info@mylfix.com",
```

Comment:
Moving this to a default would help with preventing typos such as this one. You can use the `default` class method to set your defaults.